examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/Startup.cs

```diff
@@ -51,5 +54,10 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 endpoints.MapRazorPages();
             });
         }
+
+        private void InitServices()
+        {
+            new EventHubService(Configuration);
```

Comment:
Is this in the mercy of GC to be deleted?

Reply:
Injected it instead, and warmed it up in the Configure() method.

---
In reply to: [433596607](https://github.com/Azure/AppConfiguration/pull/329#discussion_r433596607) [](ancestors = 433596607)